### DISCUSSION TITLE
Looking for a callback once a model is loaded

### DISCUSSION THREAD

**Discussion by kyadalu1** [google/filament#7247] [Q&A]
I was going through the android sample, is there a callback once a model is loaded so that i can dismiss my progress bar

**Comment by romainguy** [comment]
What kind of model do you load? glTF or something else?

**Reply by kyadalu1** [reply]
glTF

**Reply by romainguy** [reply]
If you use `AssetLoader.open()` and `ResourcesLoader.loadResources()`, loading is synchronous so there's no need for a callback. If you use an asyncLoad() in `ResourcesLoaders` you can use `asyncGetLoadProgress()` to query progress. Loading is done when progress reaches 1.0. If you use `ModelViewer` you can check the `progress` property.
